### DISCUSSION TITLE
Images aren't loading after account autoswitch from personal to free team

### DISCUSSION THREAD

**Discussion by MarisBLV** [vercel/community#6822] [Help]
### Summary

Have an issue with images after my account was switched from personal to free teams. Images are just not loading at all. In place of image I get alt text. Can anyone else have this? Can suggest any solution?

### Steps to Reproduce

No code have been changed. Only account status changed from private to free teams.

**Comment by amyegan** [comment]
Hey @MarisBLV. Can you share a link to a page where we can see the problem?
